app/Http/Controllers/WeNeedOneMoreController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+use Illuminate\Foundation\Validation\ValidatesRequests;
+use Illuminate\Routing\Controller as BaseController;
```

Comment:
we should import contracts
```suggestion
use Coinmerce\Foundation\Auth\Access\AuthorizesRequests;
use Coinmerce\Foundation\Bus\DispatchesJobs;
use Coinmerce\Foundation\Validation\ValidatesRequests;
use Coinmerce\Routing\Controller as BaseController;
```